packages/spor-theme-react/src/components/choice-chip.ts

```diff
@@ -1,20 +1,21 @@
-import { PartsStyleObject, SystemStyleObject } from "@chakra-ui/theme-tools";
+import { PartsStyleFunction, PartsStyleObject, SystemStyleFunction, SystemStyleObject } from "@chakra-ui/theme-tools";
 import { colors } from "../foundations";
 
 const choiceChipAnatomy = {
   __type: "ChoiceChip",
   keys: ["container", "label", "icon"],
 };
 
-const containerStyle: SystemStyleObject = {
+const containerStyle: SystemStyleFunction =(props)=> ({
   backgroundColor: "alias.white",
   boxShadow: `0 0 0 1px ${colors.alias.celadon}`,
   color: "alias.darkTeal",
   display: "inline-flex",
   alignItems: "center",
   fontSize: "16px",
-  transitionProperty: "box-shadow, backgroundColor",
-  transitionDuration: "fast",
+  px: 3,
+  pr: props.hasLabel ? 3 : 1,
+  pl: props.hasLabel ? 2 : 1,
```

Comment:
Du trenger ikke `px` her, siden `px` er `paddingLeft` + `paddingRight` samla. Om du overskriver begge to under, har den ikke noen effekt.

```suggestion
  pr: props.hasLabel ? 3 : 1,
  pl: props.hasLabel ? 2 : 1,
```